pages/vi/gitrepositories.md

```diff
@@ -211,6 +161,24 @@ index bf400c0..fc7380b 100644
 
 This process needs to be repeated whenever you begin to work, to make sure that you are always up to date. If there are discrepancies, it will mess up the code and you could potentially lose your saved changes, because it was not updated properly. We will provide more information on editing and saving changes in the next tutorial.
 
```

Comment:
Going along with the other comment, I think you should add back the "Summary of Steps" title and link here, just for anyone who wants to quickly go to that section.

Reply:
@BryanGazali I thought as much. Commit aa63227 fixes it.